plugins/destination/s3/client/client.go

```diff
@@ -71,7 +72,7 @@ func New(ctx context.Context, logger zerolog.Logger, spec specs.Destination) (de
 	// we want to run this test because we want it to fail early if the bucket is not accessible
 	if _, err := c.uploader.Upload(ctx, &s3.PutObjectInput{
 		Bucket: aws.String(c.pluginSpec.Bucket),
-		Key:    aws.String("/tmp/.cq-test-file"),
+		Key:    aws.String(path.Join(replacePathVariables(spec.Path, "", ""), ".cq-test-file")),
```

Comment:
It won't because I am calling `path.Clean` in `replacePathVariables` and have a test case to verify that...

Reply:
In order to use `test_table` and `test_uuid` we would also have to change the path that is being passed into the spec otherwise nothing will be replaced

Reply:
Not sure I follow what you mean? The default behavior is to add `{{TABLE}}` and `{{UUID}}` if it's not in the spec https://github.com/cloudquery/cloudquery/blob/main/plugins/destination/s3/client/spec.go#L24-L31

Reply:
Missed that... In that case will not use `""` but will use `test_table` and `test_uuid`